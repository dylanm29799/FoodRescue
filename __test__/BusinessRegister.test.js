// __tests__/BusinessRegister.test.js

import React from "react";
import BusReg from "../BusinessScreens/BusinessRegister";
import renderer from "react-test-renderer";
import { firebaseConfig } from "../config";
import * as firebase from "firebase";

firebase.initializeApp(firebaseConfig);
test("renders correctly", () => {
  const tree = renderer.create(<BusReg />).toJSON();
  expect(tree).toMatchSnapshot();
});
