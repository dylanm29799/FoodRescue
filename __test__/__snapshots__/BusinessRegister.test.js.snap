// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "height": 1334,
      "justifyContent": "flex-start",
      "width": "100%",
    }
  }
>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "alignItems": "center",
        "height": "80%",
        "justifyContent": "flex-start",
        "width": "100%",
      }
    }
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": "80%",
            "width": "100%",
          },
          undefined,
        ]
      }
    />
    <Text
      style={
        Object {
          "paddingTop": 321.42857142857144,
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Business Name"
        rejectResponderTermination={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        autoCompleteType="email"
        keyboardType="email-address"
        onChangeText={[Function]}
        placeholder="Email"
        rejectResponderTermination={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Password"
        rejectResponderTermination={true}
        secureTextEntry={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Confirm Your Password"
        rejectResponderTermination={true}
        secureTextEntry={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        keyboardType="numeric"
        onChangeText={[Function]}
        placeholder="Contact Number"
        rejectResponderTermination={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderColor": "#c44536",
          "borderRadius": 15,
          "borderWidth": 3,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "margin": 10,
          "paddingRight": 42.857142857142854,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        keyboardType="numeric"
        onChangeText={[Function]}
        placeholder="Registration Code"
        rejectResponderTermination={true}
        style={
          Object {
            "borderRadius": 20,
            "fontSize": 32.14285714285714,
            "marginRight": 21.428571428571427,
            "paddingHorizontal": 21.428571428571427,
            "paddingRight": 4.285714285714286,
            "paddingVertical": 10.714285714285714,
            "textAlign": "center",
            "width": "75%",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <Text
      style={
        Object {
          "alignItems": "center",
          "padding": 20,
        }
      }
    >
      1st time registration code - Get in contact through
      <Text
        onPress={[Function]}
        style={
          Object {
            "color": "blue",
          }
        }
      >
         
        email
      </Text>
       
      or
       
      <Text
        onPress={[Function]}
        style={
          Object {
            "color": "blue",
          }
        }
      >
        over the phone
      </Text>
    </Text>
    <Text
      onPress={[Function]}
      style={
        Object {
          "fontFamily": "MonM",
          "fontSize": 16,
          "paddingVertical": 10,
        }
      }
    >
      Pick an image
    </Text>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#c44536",
          "borderRadius": 10,
          "height": 50,
          "justifyContent": "center",
          "opacity": 1,
          "padding": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "alignItems": "center",
            "color": "#EBF38B",
            "fontFamily": "MonM",
            "fontSize": 38.57142857142857,
            "textAlign": "center",
            "width": 321.42857142857144,
          }
        }
      >
        Register
      </Text>
    </View>
    <Text />
  </View>
</View>
`;
